### DISCUSSION TITLE
scikit-rf 20 min introduction slides and short course

### DISCUSSION THREAD

**Discussion by jhillairet** [scikit-rf/scikit-rf#916] [Announcements]
The repository https://github.com/scikit-rf/scikit-rf_workshop contains materials to:
- introduce scikit-rf during a 15-20 min talk,
- give an interactive short course on scikit-rf (2 to 3 hours), to explain and learn some of the capabilities of the package.

You can use this material to communicate about scikit-rf in your company, lab or anything you want.

**Comment by denzchoe** [comment]
Thanks for this!!
